rhythm.py

```diff
@@ -280,10 +289,13 @@ def initialise_line(self, stage, user_controls_treble, start_time,
             12: 0.2
         }[self.stage]
 
+        # Either set inertia to the inertia given by the user, or calculate a good value
+        self._inertia = self._inertia_set_by_user or 0.5
```

Comment:
Does this mean the user can't set inertia to 0.0? You mention setting it to 0 in the argument's help comment.

Reply:
Oh indeed this is how python works - I didn't expect that `0.0` be coerced into `false` when evaluating the `or` operator.  The existence of `_inertia_set_by_user` doesn't make sense with the new change that actual inertia is adjusted before being used with the regression algorithm, so I've removed it in a new commit.

I do wonder, though, if the name `_inertia` is now misleading and should be changed to `_preferred_inertia` on the basis that it will?

Reply:
Done this now :)